src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -325,23 +325,35 @@ public function importModuleAction(Request $request)
         $moduleManager = $this->get('prestashop.module.manager');
         try {
             $file_uploaded = $request->files->get('file_uploaded');
-            $violations = $this->get('validator')->validate($file_uploaded, new File(
-                [
+            $notNullViolations = $this->get('validator')->validateValue(
```

Comment:
Ok, 2 pbs here:
* ``validateValue`` is deprecated since Symfony 2.5, this is why I've used ``validate()`` function before: please use ``validate()``
* also, both ``validate()`` and ``validateValue`` are able to manage .. an array of constraints ;) I know this is **still** [not documented](https://github.com/symfony/symfony-docs/pull/6584) so you have to believe dude